src/mono/netcore/System.Private.CoreLib/src/System/Reflection/CustomAttributeData.cs

```diff
@@ -158,7 +158,7 @@ public override string ToString()
 
             StringBuilder sb = new StringBuilder();
 
-            sb.Append("[" + ctorInfo.DeclaringType.FullName + "(");
+            sb.Append('[').Append(ctorInfo.DeclaringType!.FullName).Append('(');
```

Comment:
FYI, this is an implementation change but should be semantically identical, purely as an opportunistic "no need for unnecessary allocation here" thing.